### PR TITLE
=htp #1017 document marshalling behaviour of oneOf

### PR DESCRIPTION
**Pull request by ktoso**
=htp #1017 document marshalling behaviour of oneOf

### CODE REVIEW

File: akka-http/src/main/scala/akka/http/scaladsl/marshalling/Marshaller.scala [hunk 1]
```diff
@@ -102,13 +102,23 @@ object Marshaller
   /**
    * Helper for creating a "super-marshaller" from a number of "sub-marshallers".
    * Content-negotiation determines, which "sub-marshaller" eventually gets to do the job.
+   * 
+   * Please note that both marshallers will actualy be invoked in order to get the Marshalling object
```
Comment (jlprat) [inline akka-http/src/main/scala/akka/http/scaladsl/marshalling/Marshaller.scala]: Shouldn't be "all passed" instead of "both"?

Comment (Susak) [inline akka-http/src/main/scala/akka/http/scaladsl/marshalling/Marshaller.scala]: It would be more correct.

**Comment by jlprat** [comment]
I think the failure is just formatting

File: akka-http/src/main/scala/akka/http/scaladsl/marshalling/Marshaller.scala (hunk 1 above)
Comment (ktoso) [inline akka-http/src/main/scala/akka/http/scaladsl/marshalling/Marshaller.scala]: Thanks guys 👍

**Comment by ktoso** [comment]
Amended

**Pull request merged**